cli/manifest.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1" xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
+    <assemblyIdentity
+        type="win32"
+        name="Deno"
+        version="1.8.2"
```

Comment:
This is going to be a problem. Can you set it up so this is replaced in build.rs automatically by the correct version?